setup.py

```diff
@@ -791,6 +730,23 @@ def setupPackage():
             'Topic :: Scientific/Engineering :: Physics'],
         keywords=KEYWORDS,
         packages=find_packages(),
+        package_data={
```

Comment:
Is explicitly including this better than just calling `add_datafiles`? For example, if someone adds a new data file but it doesn't match one of these strings it could be hard to debug why its not being included. 

Also, did you check that everything you expect is included when ObsPy is not installed in development mode?

Reply:
I'm trying to avoid code and make the setup more declarative. Almost everything is a `*` include, so it should match anything.

However, you are right, it doesn't install everything properly.

Reply:
So there's actually a bug in `master`; some test directories do not have `__init__.py`, so they were not packaged, thus their data was not packaged. I saw the missing data first and thought that was the problem, but wasn't directly.

I realized that if we just switch to `setuptools` in one go, then `include_package_data=True` works and grabs everything in `MANIFEST.in`, so I've done that.

Now the differences are the following:

```
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: __init__.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: __pycache__
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_channel.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_compatibility.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_event.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_inventory.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_network.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_preview.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_resource_identifier.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_response.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_station.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_stats.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_stream.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_trace.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_utcdatetime.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_util_attribdict.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_util_base.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_util_decorator.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_util_misc.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_util_testing.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_util_types.py
Only in venv-modern/lib/python3.9/site-packages/obspy/core/tests: test_waveform_plugins.py
Only in venv-modern/lib/python3.9/site-packages/obspy/geodetics/tests: __init__.py
Only in venv-modern/lib/python3.9/site-packages/obspy/geodetics/tests: __pycache__
Only in venv-modern/lib/python3.9/site-packages/obspy/geodetics/tests: test_util_flinnengdahl.py
Only in venv-modern/lib/python3.9/site-packages/obspy/geodetics/tests: test_util_geodetics.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: __init__.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: __pycache__
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_backend.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_beachball.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_mopad.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_mopad_script.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_plot.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_ppsd.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_scan.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_source.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_spectrogram.py
Only in venv-modern/lib/python3.9/site-packages/obspy/imaging/tests: test_waveform.py
Only in venv-master/lib/python3.9/site-packages/obspy/io/ah: docs
Only in venv-master/lib/python3.9/site-packages/obspy/io/ascii: docs
Only in venv-master/lib/python3.9/site-packages/obspy/io/cnv: docs
Only in venv-master/lib/python3.9/site-packages/obspy/io/kinemetrics: docs
Only in venv-modern/lib/python3.9/site-packages/obspy/io/mseed/src/libmseed: LICENSE.txt
Only in venv-master/lib/python3.9/site-packages/obspy/io/mseed/src/libmseed/test: lmtestpack.c
Only in venv-master/lib/python3.9/site-packages/obspy/io/mseed/src/libmseed/test: lmtestparse.c
Only in venv-master/lib/python3.9/site-packages/obspy/io/mseed/src/libmseed/test: Makefile
Only in venv-master/lib/python3.9/site-packages/obspy/io/ndk: docs
Only in venv-master/lib/python3.9/site-packages/obspy/io/nlloc: docs
Only in venv-modern/lib/python3.9/site-packages/obspy/io/nordic/tests: __init__.py
Only in venv-modern/lib/python3.9/site-packages/obspy/io/nordic/tests: __pycache__
Only in venv-modern/lib/python3.9/site-packages/obspy/io/nordic/tests: test_nordic.py
Only in venv-master/lib/python3.9/site-packages/obspy/io/pde: docs
Only in venv-master/lib/python3.9/site-packages/obspy/io/win: docs
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: __init__.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: __pycache__
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_misc.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_ray_paths.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_seismic_phase.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_slowness_model.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_split_model.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_taup_geo.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_taup_plotting.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_tau.py
Only in venv-modern/lib/python3.9/site-packages/obspy/taup/tests: test_velocity_model.py
```

The missing `tests` directories are added, the `docs` directories are removed (which is fine because the `MANIFEST.in` means they won't be in the sdist), and I dropped a few source files from `libmseed`, since all we need there is the test data.